tests/all-spark-notebook/test_spark_notebooks.py

```diff
@@ -38,3 +38,21 @@ def test_nbconvert(container: TrackedContainer, test_file: str) -> None:
 
     expected_file = f"{output_dir}/{test_file}.md"
     assert expected_file in logs, f"Expected file {expected_file} not generated"
+
+
+def test_pandas_version(container: TrackedContainer) -> None:
```

Comment:
We don't need to repeat tests. If we have a test for some image, I made it so this test will be run for all inherited images as well.